src/core/protocol-authorization.ts

```diff
@@ -351,7 +351,8 @@ export class ProtocolAuthorization {
    * Currently the only check is: if the write is not the initial write, the author must be the same as the initial write
    * @throws {Error} if fails verification
    */
-  private static async verifyActionCondition(tenant: string, incomingMessage: RecordsRead | RecordsWrite, messageStore: MessageStore): Promise<void> {
+  private static async verifyActionCondition(
+    tenant: string, incomingMessage: RecordsRead | RecordsWrite, storageController: StorageController): Promise<void> {
```

Comment:
pretty sure that if you have to have a separate line we want all the parameters to be on their own line
``` suggestion
  private static async verifyActionCondition(
    tenant: string,
    incomingMessage: RecordsRead | RecordsWrite,
    storageController: StorageController
  ): Promise<void> {
```